tests/suites/coslite/cl.sh

```diff
@@ -0,0 +1,53 @@
+# Ensure that COS Lite deploys successfully, and that the
+# relations are setup as expected.
+run_deploy_coslite() {
+	echo
+
+	local model_name file
+	model_name="deploy-mk8s"
+	file="${TEST_DIR}/${model_name}.log"
+
+	IPADDR=$(ip -4 -j route | jq -r '.[] | select(.dst | contains("default")) | .prefsrc')
+	if [ "${IPADDR}" = "null" ]; then
+		IPADDR=$(hostname -I | awk '{print $1}')
+	fi
+	microk8s enable metallb:"$IPADDR"-"$IPADDR"
+	microk8s kubectl rollout status deployments/hostpath-provisioner -n kube-system -w
+	microk8s kubectl rollout status deployments/coredns -n kube-system -w
+	microk8s kubectl rollout status daemonset.apps/speaker -n metallb-system -w
+
+  controller_name=$(juju controllers --format json | jq -r '.controllers | keys[0]')
+	add_model "${model_name}" "${BOOTSTRAP_CLOUD}" "${controller_name}" "${file}"
+
+	juju deploy cos-lite --channel=edge --trust
+
+	wait_for "active" '.applications["alertmanager"] | ."application-status".current'
+	wait_for "active" '.applications["loki"] | ."application-status".current'
+	wait_for "active" '.applications["prometheus"] | ."application-status".current'
+	wait_for "active" '.applications["traefik"] | ."application-status".current'
+	wait_for "active" '.applications["grafana"] | ."application-status".current' 1800
+
+	# check for relations
+	wait_for 5 '.applications["alertmanager"] | ."relations" | keys | length'
+	wait_for 3 '.applications["grafana"] | ."relations" | keys | length'
+	wait_for 5 '.applications["loki"] | ."relations" | keys | length'
+	wait_for 6 '.applications["prometheus"] | ."relations" | keys | length'
+	wait_for 2 '.applications["traefik"] | ."relations" | keys | length'
```

Comment:
You might not need to query for keys, length might work on dict

Reply:
Thanks you for this. Let me try it.